src/PanZoom.js

```diff
@@ -172,6 +231,17 @@ class PanZoom extends React.Component<Props, State> {
       return false
     }
 
+    if (this.wheelPanning) {
```

Comment:
This section handles starting a mouse-click pan while already panning with the trackpad. It will basically stop the panning that happens with the `wheel` events, and then start panning with the mouse. The goal is to have a smooth handoff between the two styles of panning.

Reply:
Nice!